op-node/rollup/engine/build_seal.go

```diff
@@ -106,10 +107,17 @@ func (eq *EngDeriver) onBuildSeal(ev BuildSealEvent) {
 	eq.metrics.RecordSequencerBuildingDiffTime(buildTime - time.Duration(eq.cfg.BlockTime)*time.Second)
 
 	txnCount := len(envelope.ExecutionPayload.Transactions)
```

Comment:
Its possible that a helper function on the envelope or execution payload could dedupe code, i feel like we do deposit count elsewhere in the codebase, but also it may be on different data structures

Reply:
I found one inside this package and used it